### PR TITLE
added example

### PR DESCRIPTION
**Pull request by gkteco**
Example of authenticating with `iridium-rust-client` exmaple

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+/target
+
```
Comment (joshfischer1108) [inline .gitignore:2]: `/target` is specific to Java.  Let's use a `.gitignore` similar to [this](https://github.com/intellij-rust/intellij-rust/blob/master/.gitignore)

File: .idea/.gitignore
Comment (joshfischer1108) [inline .idea/.gitignore:1]: We need to delete all of the files in the `.idea` from git history before merging. [Something like this should do the trick](https://stackoverflow.com/a/52643437)

File: src/main.rs [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+                }
+
+            } else {
+                eprintln!("Error handling callback");
+                Err(warp::reject())
+            }
+
+        }
+    });
+
```
Comment (joshfischer1108) [inline src/main.rs:39]: Is this a Web App to render static files that are presented in a browser or a backend application that would act as a remote server?

**Review by joshfischer1108 — COMMENTED** [review]
Thanks for this @gkteco , I left a few comments.

File: .env [hunk 3]
```diff
@@ -0,0 +1,4 @@
+RUST_IRIDIUM_BASE_URL = "https://test.iridium.software/"
+RUST_PUBLIC_IRIDIUM_REDIRECT_URI = "http://localhost:8080/callback"
+RUST_PUBLIC_IRIDIUM_CLIENT_ID = "26bf7b9643282e2eea9f3d4884381db7"
```
Comment (joshfischer1108) [inline .env:3]: You prob don't want this `RUST_PUBLIC_IRIDIUM_CLIENT_ID` value in git.

Comment (gkteco) [reply .env:3]: yup, thats why I closed this PR. Addressing this.

File: .gitignore (hunk 1 above)
Comment (gkteco) [reply .gitignore:2]: Yup, I can do this.

File: .idea/.gitignore
Comment (gkteco) [reply .idea/.gitignore:1]: ok 👍

File: src/main.rs (hunk 2 above)
Comment (gkteco) [reply src/main.rs:39]: This is a web app that will render static files that are presented in a browser.

**Pull request closed without merging**